backend/internal/handlers/classrooms/assignments/works/filetree.go

```diff
@@ -0,0 +1,43 @@
+package works
+
+import (
+	"errors"
+	"net/http"
+
+	"github.com/CamPlume1/khoury-classroom/internal/errs"
+	"github.com/gofiber/fiber/v2"
+)
+
+func (s *WorkService) GetFileTree(c *fiber.Ctx) error {
+	work, err := getWork(s, c)
+	if err != nil {
+		return c.SendStatus(404)
+	}
+	if work.RepoName == nil || work.SubmittedPRNumber == nil {
+		return errs.BadRequest(errors.New("work has not been submitted for grading yet"))
+	}
+
+	tree, err := s.githubappclient.GetFileTree(work.OrgName, *work.RepoName, *work.SubmittedPRNumber)
+	if err != nil {
+		return err
+	}
+
+	return c.Status(http.StatusOK).JSON(tree)
```

Comment:
can we add titles to both of these methods' returns for consistency

Reply:
only for the tree since blob needs to return raw contents